src/schema/types/mutations/Portfolio.ts

```diff
@@ -1,15 +1,31 @@
 import { arg, nonNull, intArg } from "nexus";
 
+import { context } from "../../../context";
+import {
+  getPortfolioByUser,
+  modifyPortfolio,
+  getLikePortfolio,
+  createLikePortfolio,
+  deleteLikePortfolio,
+} from "service/portfolio";
+
```

Comment:
매번 함수가 추가될 때마다 추가하는 것보단, 하나의 객체로 불러와서 사용하는 것도 좋아보여요. 그럼 함수명이나 변수명이 겹칠 일이 줄어들고, 어떤 역할의 함수인지 파악하는 것도 한결 편해질 것 같네요 :D
```suggestion
import { PortfolioService } from "service";

```

Reply:
이렇게 바꾸면 아래의 함수들을 사용한 부분에서 살짝씩 수정 해줘야 겠네요